src/test/test-strv.c

```diff
@@ -656,6 +658,10 @@ static void test_strv_make_nulstr_one(char **l) {
         assert_se(strv_make_nulstr(q, &c, &m) >= 0);
         assert_se(m == n);
         assert_se(memcmp(b, c, m) == 0);
+
+        NULSTR_FOREACH(s, b)
+                assert_se(streq(s, l[i++]));
+        assert_se(i == strv_length(l));
```

Comment:
It doesn't matter for the test binaries, but these expressions do not have side effects. So in theory, the `_se` could be dropped. But that's just nitpicking.